docs/modules/management/pages/license-info.adoc

```diff
@@ -58,7 +58,7 @@ Its output is similar to the following:
 request through the REST API (if enabled; see the xref:management:rest-endpoint-groups.adoc[Using the REST Endpoint Groups section]) on the `/license` as shown below:
 
 ```
-curl --data "${CLUSTERNAME}&$\{PASSWORD}&${LICENSE}" http://localhost:5001/hazelcast/rest/license
```

Comment:
this is not the `[source]` section, but it seems it's should not be there anyway. 
this is how it's currently rendered:
![image](https://user-images.githubusercontent.com/158619/128868368-060ddc79-7ce5-4c07-84c4-7bbb46c4a4d4.png)

Reply:
probably the triple backticks have the same effect as `source`
(yes, I know nothing about AsciiDoc)